pulsar-common/src/main/java/org/apache/pulsar/policies/data/loadbalancer/LocalBrokerData.java

```diff
@@ -235,7 +235,16 @@ private void updateBundleData(final Map<String, NamespaceBundleStats> bundleStat
     }
 
     public double getMaxResourceUsage() {
-        return max(cpu.percentUsage(), memory.percentUsage(), directMemory.percentUsage(), bandwidthIn.percentUsage(),
+        return getMaxResourceUsage(true);
+    }
+
+    public double getMaxResourceUsage(boolean checkMemoryUsage) {
+        if (checkMemoryUsage) {
```

Comment:
I would suggest to remove this config option and instead only check the direct memory but not the heap memory usage here.

Reply:
Updated.